packages/core/stories/elements/Button/Button.stories.mdx

```diff
@@ -12,160 +12,153 @@ import * as stories from './Button.stories.js';
 />
 
 # Buttons
-Buttons express what action will occur when the user clicks or touches it. Buttons are used to initialize an action, either in the background or foreground of an experience.
-<Canvas withSource="none" >
+
+Buttons are used in forms and interactive applications, but also as prominent call-to-action links on website pages.
```

Comment:
So here I'm thinking whether we should make the distinction between buttons and links. In the Link doc, there's a variant called "Link styled as a button", meaning button "styles" are allowed to be used as links (whether or not it's a good practice it's up for debate).  Technically, a button should only be used to trigger actions vs links are used for navigation. 
```suggestion
Buttons are used in forms and interactive applications to trigger actions.

Reply:
What I'm trying to say here is whether this doc is about the element button or the styles of buttons — if we are keeping "button" just as the element itself, then I think to distinguish it from links will make more sense.